src/pages/Booking/Seats/Seats.js

```diff
@@ -0,0 +1,388 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import PeopleCount from './PeopleCount';
+import resetIcon from './images/icon_reset.png';
+
+function Seats() {
+  const [movieData, setMovieData] = useState([]);
+  const [peopleTotalCount, setPeopleTotalCount] = useState({
+    성인: 0,
+    대학생: 0,
+    청소년: 0,
+    우대: 0,
+  });
```

Comment:
아래 리셋 함수에서 초기값을 다시 부여하고 있는 로직이 있는 것이 있으니
초기값을 따로 변수로 만들어서 사용하는 것이 코드의 중복을 피하는 방법이라고 보여집니다.

```suggestion
  const initCountValue = {
    성인: 0,
    대학생: 0,
    청소년: 0,
    우대: 0,
  }

  const [peopleTotalCount, setPeopleTotalCount] = useState(initCountValue);
```
위의 형태로 만들고 아래 reset에서도 쓸 수 있도록 해주세요